cli/smartcontract/smart_contract.go

```diff
@@ -69,6 +69,10 @@ var (
 	}
 )
 
+// ModVersion contains `pkg/intepop` module version
+// suitable to be used in go.mod.
+var ModVersion string
```

Comment:
Give it some default, like "latest"?

Reply:
We can't do it here because of how `-X` flag works, fixed in the func itself.